### PR TITLE
Bz2 381/snackbar

### PR DESCRIPTION
**Pull request by Arcoders**

### CODE REVIEW

File: sass/module/_snackbar copy.scss [hunk 1]
```diff
@@ -0,0 +1,97 @@
+@mixin applyPosition (
```
Comment (adam-dilibero) [inline sass/module/_snackbar copy.scss]: What is this file? seems like a duplicate of _snackbar.scss?

File: sass/_vars.scss [hunk 2]
```diff
@@ -69,7 +69,6 @@ $purple:                  #9c27b0;
 $indigo:                  #9c27b0;
 $teal:                    #009688;
 $yellow:                  #ffc107;
-$cyan:                    #00bcd4;
```
Comment (adam-dilibero) [inline sass/_vars.scss:72]: Is this variable no longer used?

**Review by adam-dilibero — CHANGES_REQUESTED** [review]

File: sass/module/_snackbar copy.scss (hunk 1 above)
Comment (Arcoders) [reply sass/module/_snackbar copy.scss]: You're right, it's duplicated, I've deleted it

File: sass/_vars.scss (hunk 2 above)
Comment (Arcoders) [reply sass/_vars.scss:72]: I deleted it because it already exists

**Review by adam-dilibero — APPROVED** [review]

**Review by suzanneb9 — APPROVED** [review]

**Pull request merged**